### DISCUSSION TITLE
help with downloading

### DISCUSSION THREAD

**Discussion by GFR3-404NOtfOuND** [zencq/NomNom#60] [Q&A] [answered]
i cant fidnd the .exe file.. is it my computer?

**Comment by cengelha** [comment, marked as answer]
Hi. You prolly downloaded the source code. Please go to https://github.com/cengelha/NomNom/releases and download "NomNom.zip" from the assets of the latest release

**Reply by GFR3-404NOtfOuND** [reply]
oki, thanks so much~!
